app/_src/guides/gateway-api.md

```diff
@@ -0,0 +1,336 @@
+---
+title: Kubernetes Gateway API
+---
+
+To get traffic from outside your mesh inside it (North/South) with {{site.mesh_product_name}} you can use
+a builtin gateway.
+
+In the [quickstart](/docs/{{ page.version }}/quickstart/kubernetes-demo/), traffic was only able to get in the mesh by port-forwarding to an instance of an app
+inside the mesh.
+In production, you typically set up a gateway to receive traffic external to the mesh.
+In this guide you will add [a built-in gateway](/docs/{{ page.version }}/using-mesh/managing-ingress-traffic/builtin/) in front of the demo-app service and expose it publicly.
+We will deploy and configure Gateway using [Kubernetes Gateway API](https://gateway-api.sigs.k8s.io/).
+
+Service graph of the demo app with a builtin gateway on front:
+{% mermaid %}
+flowchart LR
+  subgraph edge-gateway
+    gw0(/ :8080)
+  end
+  demo-app(demo-app :5000)
+  redis(redis :6379)
+  gw0 --> demo-app
+  demo-app --> redis
+{% endmermaid %}
+
+## Prerequisites
+
+- Completed [quickstart](/docs/{{ page.version }}/quickstart/kubernetes-demo/) to set up a zone control plane with demo application
+
+## Install Gateway API CRDs
+
+To install Gateway API please refer to [official installation instruction](https://gateway-api.sigs.k8s.io/guides/#install-standard-channel).
+
+You also need to manually install {{site.mesh_product_name}} [GatewayClass](https://gateway-api.sigs.k8s.io/api-types/gatewayclass/):
```

Comment:
this isn't true if you install the CRDs first

Reply:
yes, but this guide assumes that user followed quickstart and already have control plane and demo installed